### PR TITLE
VIX-3372 Migrate QuickFont, SharpFont to .NET 6 and OpenTK 4.7.5

### PR DESCRIPTION
**Pull request by johncbaur**
VIX-3372 Migrate QuickFont, SharpFont to .NET 6 and OpenTK 4.7.5

### CODE REVIEW

File: src/Vixen.Modules/Editor/FixtureGraphics/OpenGL/MovingHeadOpenGL.cs [hunk 1]
```diff
@@ -671,9 +671,9 @@ public void Initialize(double length, double maxBeamLength, double beamTranspare
...
 				
 				// Draw the text
-				// TODO .NET 6 commented _qFontDrawing.Print(_qFont, MovingHead.Legend, positionOfText, QFontAlignment.Left, MovingHead.LegendColor);
+				_qFontDrawing.Print(_qFont, MovingHead.Legend, positionOfText, QFontAlignment.Left, MovingHead.LegendColor);
 				_qFontDrawing.RefreshBuffers();
-				// TODO .NET 6 commented _qFontDrawing.ProjectionMatrix = viewMatrix * projectionMatrix;				
+				_qFontDrawing.ProjectionMatrix = viewMatrix * projectionMatrix;				
 				_qFontDrawing.Draw();
 			}
 		}
```
Comment (jeffu231) [inline src/Vixen.Modules/Editor/FixtureGraphics/OpenGL/MovingHeadOpenGL.cs:679]: Did you also look at line 703 QFontDrawing.DisposeStaticState() that I commented also? If it is not needed then we can remove the line.

**Review by jeffu231 — COMMENTED** [review]
Just wanted to be sure you saw the other line I commented.

**Comment by johncbaur** [comment]
Good catch.  I think that line is necessary.  Updated PR.

File: src/Vixen.Modules/Editor/FixtureGraphics/OpenGL/MovingHeadOpenGL.cs (hunk 1 above)
Comment (jeffu231) [reply src/Vixen.Modules/Editor/FixtureGraphics/OpenGL/MovingHeadOpenGL.cs:679]: I don't see an updated PR yet.

**Comment by johncbaur** [comment]
Sorry...should be there now.

**Pull request merged**